src/main/java/pixelperfect/Game.java

```diff
@@ -33,7 +33,7 @@ public static void main(String[] args) {
    */
   @Override
   public void simpleInitApp() {
-    spaceship = new Spaceship();
+		spaceship = Spaceship.getInstance();
```

Comment:
I understand that though, I envision the Spaceship to be of a similar nature. However, just because a model class will probably only be instantiated once during the process, does not mean it is a good candidate for the Singleton design pattern. If you would think this way you could make half of the classes a Singleton. When it concerns a class that would otherwise be a Utility class (e.g. not object oriented, using static methods) which will be used by clients all over the place, you might have a reason to do this. This way some global data / state will be shared among all clients. I do not agree that this holds for the Spaceship.